src/components/Input.js

```diff
@@ -0,0 +1,157 @@
+    } else {
+      setError('Country not found');
+      setTimeout(() => {
+        setError('');
+      }, 3000);
+    }
+  };
+
+  const handleArrowLeftClick = () => {
+    setShowSearchResult(false);
+  };
+
+  return (
+    <>
+      <p className="logo">
+        {' '}
+        <span><FaVirus /></span>
+        {' '}
+        Covid-19 Tracker
+      </p>
+      <div className="navbar">
+        <section>
+          <Link to="/" className="left-arrow">
+            <FaChevronLeft onClick={handleArrowLeftClick} />
+          </Link>
+        </section>
+        <form className="search-holder" onSubmit={handleSubmit}>
+          <input
+            type="text"
+            placeholder="search country"
+            className="search-item"
+            value={search}
+            onChange={searchCountry}
+          />
+          <button type="submit" className="submit-search">
+            <FaSearch />
+          </button>
+        </form>
+        <div>
+          <FaMicrophone className="microphone" />
+          <RiSettings2Fill />
+        </div>
+      </div>
+      {error && <p className="error">{error}</p>}
+      {showSearchResult && filteredData
+        && (
+        <section>
+          <h2 className="Info title">
+            {' '}
+            It locates in the
+            {filteredData.continent}
+            {' '}
+            continent, here is the data:
+          </h2>
+          <div className="details-container">
+            <div className="detail-holder">
+              <span className="icon" style={icon}>
+                <FaSkull />
+              </span>
+              <p>
+                Death(s) :
+                {' '}
+                {filteredData.deaths}
+              </p>
+            </div>
+            <div className="detail-holder">
+              <span className="icon" style={icon}>
+                <FaHeartbeat />
+              </span>
+              <p>
+                Recovered :
+                {' '}
+                {filteredData.recovered}
+              </p>
+            </div>
+            <div className="detail-holder">
+              <span className="icon" style={icon}>
+                <FaPeopleArrows />
+              </span>
+              <p>
+                Cases :
+                {' '}
+                {filteredData.cases}
+              </p>
+            </div>
+            <div className="detail-holder">
+              <span className="icon" style={icon}>
+                <FaHeadSideCough />
+              </span>
+              <p>
+                Infected :
+                {' '}
+                {filteredData.active}
+              </p>
+            </div>
+            {/* <div className="detail-holder">
+              <span className="icon" style={icon}>
+                <FaThermometerThreeQuarters />
+              </span>
+              <p>
+                Tests:
+                {' '}
+                {filteredData.tests}
+              </p>
+            </div> */}
```

Comment:
- [ ] Please remove this commented code to ensure that our code looks cleaner and more professional. Leaving commented code in our codebase is not a good practice. :ok_hand: 😊